piker/data/_sampling.py

```diff
@@ -233,16 +238,23 @@ async def sample_and_broadcast(
             # thus other consumers still attached.
             subs = bus._subscribers[sym.lower()]
 
+            lags = 0
             for (stream, tick_throttle) in subs:
 
                 try:
-                    if tick_throttle:
-                        # this is a send mem chan that likely
-                        # pushes to the ``uniform_rate_send()`` below.
-                        await stream.send(quote)
-
-                    else:
-                        await stream.send({sym: quote})
+                    with trio.move_on_after(0.2) as cs:
```

Comment:
This is effectively a backpressure detector and should basically never get entered unless there's an issue on the consumer side (which hopefully all these are resolved now).

Not sure if we should put this onto master though with the `.breakpoint()` but I haven't seen it happen yet in practice.